app/src/main/java/sitsko/vlad/epamhomework/fragments/SecondFragment.java

```diff
@@ -0,0 +1,20 @@
+package sitsko.vlad.epamhomework.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import sitsko.vlad.epamhomework.R;
+
+public class SecondFragment extends Fragment {
```

Comment:
looks like FirstFragment and SecondFragment could be combined to one (something like CombinedFragment) and just sent text as argument.